TERIMLERSOZLUGU.md

```diff
@@ -32,19 +32,19 @@ Terimler sözlüğü ayrıca Türkçe'ye çevirilmeyecek sözcüklerin de belirl
 [z](#z) -
 
 #A
- * Arrow Functions:
- * Assertion: Kontrol fonksiyonu
+ * Arrow Functions: Ok Fonksiyonları
+ * Assertion: Kontrol Fonksiyonu
 
 #B
- * Build: Yapı
+ * Build: İnşa Etmek

```

Comment:
Derleme'de denilebilir. Yapı ve İnşa Etmek bana inşaat terimlerini çağrıştırıyor.